coda/coda_mdstore/urls.py

```diff
@@ -11,6 +11,10 @@
         r'^bag/(?P<identifier>ark:\/\d+\/.+?)/links/$', views.bagURLList,
         kwargs={'html': True}, name='bag-links'
     ),
+    re_path(
+        r'^bag/(?P<identifier>ark:\/\d+\/.+?).zip$', views.bagURLList,
```

Comment:
Is this regex correct? I feel like the `.+?` part is odd. Should it just be `.*`?

Reply:
or just `.+` without the `?` because you would expect it to have atleast one char there.